README.md

```diff
@@ -137,7 +137,9 @@ For this to work, you need to generate a search database with format `js`, and
 then add to every call of `odoc html-generate` the flags `--search-uri
 sherlodoc.js --search-uri db.js`. `sherlodoc.js` is installed in your path by
 opam, but `db.js` is the search database you generate and can be renamed as you
-wish.
+wish. After than you should copy the js files in the right spot. You do need to
+have them in correct spot before calling odoc, it does not check if the files
+exist.
```

Comment:
"The correct spot" is the path specified in the `--search-uri` argument relative to the output directory. So in your example, the root of the `--output` directory.

I don't know what you mean with "After that" and "before calling odoc" but there is no constraint on the order!

Also, the mention of the fact that "sherlodoc.js" is installed in the path by opam is still confusing!

> Be sure to copy the two js files in the output directory given to the html-generate command.

Reply:
> Also, the mention of the fact that "sherlodoc.js" is installed in the path by opam is still confusing!
Yes this is confusing, I will look for a better way to explain it.